### PR TITLE
add support for debian's /etc/issue

### PR DESCRIPTION
**Pull request by ghedamat**
extract parsing logic for issue file into conf and add tests as well

/cc @phillmv

### CODE REVIEW

**Comment by ghedamat** [comment]
(wait for merge :) )

File: agent/conf.go [hunk 1]
```diff
@@ -53,6 +52,20 @@ func NewConfFromEnv() *Conf {
 	return conf
 }
 
+func (conf *ServerConf) ParseDistro() {
+	if conf.DistroString == "" || conf.DistroString == "unknown" {
+		// We can find out distro and release on debian systems
+		etcIssue, err := ioutil.ReadFile(env.DistributionFile)
```
Comment (ghedamat) [inline agent/conf.go]: rename this and ENV variable to `debianLikeFile`

**Comment by phillmv** [comment]
superseded with #110 - sorry Mattia :P

**Comment by ghedamat** [comment]
:D seems good

**Pull request closed without merging**